lib/widgets/conversations/add_conversation_widget.dart

```diff
@@ -13,26 +12,21 @@ class AddConversationWidget extends StatelessWidget {
 
   AddConversationWidget({this.conversationData, this.user, this.conversation});
 
-  void addConversation(Conversations conversationData, User user, String name, ctx) {
+  void addConversation(
```

Comment:
This looks strange to me, is it due to a linter?

Reply:
It is, right now I'm using the flutter vs code extension to do the linting, in this case it wrapped the function because the line was too long